distribution/client/src/lib/update.ts

```diff
@@ -48,6 +50,12 @@ export default class Update {
     } else {
       this.healthChecker = new HealthChecker(process.env.JENKINS_URL || 'http://127.0.0.1:8080');
     }
+
+    if (this.options.status) {
+      this.status = this.options.status;
+    } else {
+      this.status = this.status = new Status(this.app, { flavor: process.env.FLAVOR });
```

Comment:
`this.status = this.status = new Status`

is the redundancy important here? Or is this just a typo?

Reply:
:man_facepalming: :-), the latter